release_dashboard/test/fakes/services/fake_conductor.dart

```diff
@@ -73,4 +73,10 @@ class FakeConductor extends ConductorService {
   ConductorState? get state {
     return testState;
   }
+
+  @override
+  get engineCheckoutDirectory => '${rootDirectory.path}/flutter_conductor_checkouts/engine';
```

Comment:
do you have to hardcode `flutter_conductor_checkouts`, or is this available as a variable in the UI?

Reply:
also, definitely the repo name is configurable too

Reply:
oh wait, this is the fake_conductor. maybe it doesn't matter?

Reply:
yes, it is the fake, we could pass in whatever string.